### PR TITLE
Repropose support cronjob

### PR DESCRIPTION
**Pull request by TommyLike**
Propose to support cronjob in volcano. Spec as below.

### CODE REVIEW

File: docs/design/cronjob.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# CronJob
+
+[@TommyLike](https://github.com/tommyLike/); April 23, 2019
+
+## Motivation
+
+`CronJob` is designed to help end user to schedule their batch job at some specific time, such as if you want to
+schedule a batch job at a low activity period, or just need the job to be executed periodically, it's especially
```
Comment (hzxuzhonghu) [inline docs/design/cronjob.md]: I can see the value of delay scheduling/running, but can hardly imagine when  periodically run needed. Can you elaborate a little?

Comment (TommyLike) [reply docs/design/cronjob.md]: It's a common case for high performance tasks which usually task a long time, for example, daily processing business data, generate report, update warehouse.

Comment (xieydd) [reply docs/design/cronjob.md]: CronJob is valueable, add the flexibility of scheduler.

Comment (k82cn) [reply docs/design/cronjob.md]: yes, we need this feature :)

**Comment by TommyLike** [comment]
@k82cn what's the next step of this proposal?

**Comment by k82cn** [comment]
it's low priority for now, let's handle this in next release.

**Comment by TommyLike** [comment]
@k82cn  close and open when next release.

**Pull request closed without merging**